projects/plugins/jetpack/tests/php/modules/publicize/Publicize_Test.php

```diff
@@ -251,9 +251,8 @@ public function test_publicize_post_type_is_publicizeable_cpt() {
 	}
 
 	public function test_publicize_get_all_connections_for_user() {
-		if ( ! defined( 'JETPACK_SOCIAL_USE_ADMIN_UI_V1' ) ) {
-			define( 'JETPACK_SOCIAL_USE_ADMIN_UI_V1', true );
-		}
+		$this->markTestSkipped();
```

Comment:
Why do we skip this? We could also remove it if it's not relevant anymore

Reply:
Yeah, it's no longer relevant with the connections format as it was updated during the connections schema changes. 

Should we remove it then?

Reply:
Right.